liver_imaging_analysis/engine/engine.py

```diff
@@ -56,6 +58,11 @@ def __init__(self):
         self.postprocessing_transforms = self.get_postprocessing_transforms(
              config.transforms["post_transform"]
         )
+        
+        self.load_data()
```

Comment:
What if you intend to use the instantiated model for prediction only (in the user interface for example)? why would loading training and testing sets be necessary? What if training and testing sets don't exist? Will this still run?

Reply:
yes you are right, I misunderstood a previous comment.